### PR TITLE
Resolving issues in base.html and course_list.html

### PR DESCRIPTION
**Pull request by AdityaAshvin**

### CODE REVIEW

**Comment by AdityaAshvin** [comment]
ss

**Comment by AdityaAshvin** [comment]
Changes made :
1) Removed the login and signup modals.
2) when login is pressed in the dropdown menu the user is redirected to the login page which was already being used.
3) when signup is pressed in the dropdown menu the user is redirected to the signup page which was already being used.
4) Navbar is removed out of the div container

**Comment by Tanuj22** [comment]
Hey @AdityaAshvin everything looks fine here. You need to remove the merge conflicts. Please remove those. If you are having any trouble removing the conflicts please ping me

**Comment by AdityaAshvin** [comment]
@Tanuj22 conflicts are resolved. @abdulbasith007 please review and merge this PR.

File: courseify/courses/templates/courses/base.html [hunk 1]
```diff
@@ -1,11 +1,28 @@
 <!-- {% load static %} -->
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< HEAD
```
Comment (Tanuj22) [inline courseify/courses/templates/courses/base.html]: @AdityaAshvin These are due to conflicts.

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
The conflicts aren't revolved yet

File: courseify/courses/templates/courses/base.html (hunk 1 above)
Comment (AdityaAshvin) [reply courseify/courses/templates/courses/base.html]: okay. I'll fix them.

**Comment by AdityaAshvin** [comment]
@Tanuj22 Mentioned changes have been made.

**Pull request merged**